dag_in_context/src/optimizations/passthrough.egg

```diff
@@ -0,0 +1,47 @@
+;; Pass through thetas
+(rule ((= lhs (Get loop i))
```

Comment:
For non-terminating loops, this is unsound. We should have a test for this
I think you need to check the type of the state edge, and don't do this optimization if it's a state edge.

Reply:
Hmm. I don't need this for my optimization. Since we're in hack mode, should I just comment this one out with a comment?

Reply:
I went ahead and did this to move faster. Feel free to unresolve if you want me to do something else.